.editorconifg

```diff
@@ -0,0 +1,14 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+
+# tabstops for .c .cpp and .h files
+[*.{c,cpp,h}]
+indent_style = tab
```

Comment:
A lot of code assumes 4-width tab spacing for alignment purposes, so maybe having `indent_size = 4` would be useful here too

Reply:
I actually prefer 4 space tab stops personally, but parsec's code seems to use tabs instead of space based tab stops.   4 space tabs is more widely used, but that may be another one of those "find and replace wholesale" things.  I was just trying to come up with something so that issues like the tab stop issue in one of the latest PR doesn't happen again.